script.js

```diff
@@ -60,5 +60,17 @@ const checkKey = (event) => {
     }
 }
 
+    // btn onclick events
```

Comment:
Please fix indentation. By the way, prefer using `() => {}` instead `function(){}`.

Reply:
ok, will update